templates/upload_image.html

```diff
@@ -0,0 +1,24 @@
+<div class="modal fade" id="uploadImageModal" tabindex="-1" role="dialog" aria-labelledby="uploadImageModalLabel" aria-hidden="true">
+<div class="modal-dialog" role="document">
+    <div class="modal-content">
+    <div class="modal-header">
+        <h5 class="modal-title" id="uploadImageModalLabel">Upload Image</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+        <span aria-hidden="true">&times;</span>
+        </button>
+    </div>
+    <form action="/images/" method="POST" enctype="multipart/form-data">
+        {% csrf_token %}
+        <div class="modal-body">
+            <input type="hidden" name="image-upload-type" id="image-upload-type" />
```

Comment:
`<input>` elements don't need the **_XHTML_** closing `.../>` tag.

Reply:
Removed the closing tag.